src/nft/components/explore/Banner.tsx

```diff
@@ -0,0 +1,185 @@
+import { ReactNode, useEffect, useState } from 'react'
+import { useQuery } from 'react-query'
+import { Link } from 'react-router-dom'
+
+import * as styles from './Banner.css'
+
+const Banner = () => {
+  /* Sets initially displayed collection to random number between 0 and 4  */
+  const [current, setCurrent] = useState(Math.floor(Math.random() * 5))
+  const [hovered, setHover] = useState(false)
+  const { data: collections } = useQuery(
+    ['trendingCollections'],
+    () => {
+      return fetchTrendingCollections({ volumeType: 'eth', timePeriod: TimePeriod.OneDay, size: 5 })
+    },
+    {
+      refetchOnReconnect: false,
+      refetchOnWindowFocus: false,
+      refetchOnMount: false,
+    }
+  )
+
+  useEffect(() => {
+    /* Rotate through Top 5 Collections on 15 second interval */
+    const interval = setInterval(async () => {
+      if (collections && !hovered && !stale) {
+        const nextCollectionIndex = (current + 1) % collections.length
+        setCurrent(nextCollectionIndex)
+      }
+    }, 15_000)
+    let stale = false
+    return () => {
+      stale = true
+      clearInterval(interval)
+    }
+  }, [current, collections, hovered])
+
+  return (
+    <Box
+      className={styles.fullWidth}
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      cursor="pointer"
+    >
+      {collections && collections[current] ? (
+        <Link to={`/nfts/collection/${collections[current].address}`} style={{ textDecoration: 'none' }}>
+          <Box style={{ height: '386px' }}>
+            <CollectionWrapper bannerImageUrl={collections[current].bannerImageUrl}>
+              <Box
+                as="section"
+                className={section}
+                display="flex"
+                flexDirection="row"
+                flexWrap="nowrap"
+                paddingTop="40"
+                position="relative"
+              >
+                <CollectionDetails collection={collections[current]} hovered={hovered} rank={current + 1} />
+              </Box>
+              <Center marginTop="16" position="relative">
+                {Array(collections.length)
+                  .fill(null)
+                  .map((value, carouselIndex) => (
+                    <CarouselIndicator
+                      active={carouselIndex === current}
+                      onClick={() => setCurrent(carouselIndex)}
+                      key={carouselIndex}
+                    />
+                  ))}
+              </Center>
+            </CollectionWrapper>
+          </Box>
+        </Link>
+      ) : (
+        <>
+          {/* TODO: Improve Loading State */}
+          <p>Loading</p>
+        </>
+      )}
+    </Box>
+  )
+}
+
+export default Banner
+
+/* Collection Wrapper: applies background image to entire banner */
+const CollectionWrapper = ({ bannerImageUrl, children }: { bannerImageUrl: string; children: ReactNode }) => (
```

Comment:
The CollectionWrapper should be a part of the CollectionDetails. Keeping it separate only complicates the markdown for the Banner.